db/migrate/20201001134619_remove_not_null_constraint_from_mapping_terms_predicate_field.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class RemoveNotNullConstraintFromMappingTermsPredicateField < ActiveRecord::Migration[6.0]
+  def change
+    change_column :mapping_terms, :predicate_id, :integer, null: true
```

Comment:
Shouldn't the column's type remain `bigint`? It's has an FK constraint, and the `predicates` table's `id` is of the `bigint` type, so those need to match.